components/published-toggle/index.jsx

```diff
@@ -29,7 +29,7 @@ const PublishedToggle = ({ defaultVisibility, isExpanded }) => {
         }}
         checked={isChecked}
       />
-      <div className={styles.toggleSwitch}>
+      <div className={classNames(styles.toggleSwitch, className)}>
```

Comment:
I had a thought about this and maybe we should start using only classNames from @oacore/design

Reply:
I was too lazy to update it. If you want to I will do this change 🙂

Reply:
No need, I will do it for all occurrences in one of my commits.